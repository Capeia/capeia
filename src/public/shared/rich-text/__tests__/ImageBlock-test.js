// @flow
/* eslint-env jest, jasmine */

import React from 'react'
import renderer from 'react-test-renderer'

describe('<ImageBlock>', () => {
  it('renders correctly', () => {
    const block = {
      key: '1',
      depth: 0,
      type: 'IMAGE',
      text: 'foo bar baz',
      data: {
        url: 'http://example.com/meme.jpg'
      },
      entityRanges: [],
      inlineStyleRanges: []
    }

    jest.deepUnmock('../ImageBlock')
    const Component = require('../ImageBlock').default
    const tree = renderer.create(<Component block={block} entityMap={{}} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
