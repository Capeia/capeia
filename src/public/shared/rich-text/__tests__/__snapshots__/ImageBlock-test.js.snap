// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImageBlock> renders correctly 1`] = `
<div
  className="articleMediaBlock"
>
  <div
    className="container"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <img
      src="http://example.com/meme.jpg"
    />
  </div>
  <div
    className="description"
  >
    <span
      style={Object {}}
    >
      foo bar baz
    </span>
  </div>
</div>
`;
